server/views/components/sample/sample.njk

```diff
@@ -2,6 +2,6 @@
   <div class="sample-component">
     <h2 class="sample-component__heading">Sample component</h2>
     <p>{{ text }}</p>
-    {% include "views/components/grid-placeholder.njk" %}
+    {% include "views/grid/placeholder/placeholder.njk" %}
```

Comment:
Just with how we include them elsewhere, I am working on a component thing for nunjucks as we type.

If we go with this - it would be nice to know if we can add JS etc to the `README`s, as [mentioned here](https://github.com/wellcometrust/wellcomecollection.org/pull/53#discussion_r84868916) we could do some interesting component tracking making sure we're not keeping or enhancing components that aren't performing/